packages/quantic/cypress/integration/standalone-search-box/standalone-search-box.cypress.ts

```diff
@@ -0,0 +1,127 @@
+import {configure} from '../../page-objects/configurator';
+import {
+  InterceptAliases,
+  interceptSearch,
+  routeMatchers,
+} from '../../page-objects/search';
+import {StandaloneSearchBoxExpectations as Expect} from './standalone-search-box-expectations';
+import {StandaloneSearchBoxActions as Actions} from './standalone-search-box-actions';
+import {scope} from '../../reporters/detailed-collector';
+
+interface StandaloneSearchBoxOptions {
+  placeholder: string;
+  withoutSubmitButton: boolean;
+  numberOfSuggestions: number;
+  redirectUrl: string;
+}
+
+function mockSuggestions() {
+  cy.intercept(routeMatchers.querySuggest, (req) => {
+    req.continue((res) => {
+      res.body.completions = [
+        {
+          expression: 'test',
+          highlighted: '[test]',
+        },
+        {
+          expression: 'test 2',
+          highlighted: '[test][2]',
+        },
+      ];
+      res.body.responseId = 21322434;
+      res.send();
+    });
+  }).as(InterceptAliases.QuerySuggestions.substring(1));
+}
+
+describe('quantic-standalone-search-box', () => {
+  const standaloneSearchBoxUrl = 's/quantic-standalone-search-box';
+
+  function visitStandaloneSearchBox(
+    options: Partial<StandaloneSearchBoxOptions> = {}
+  ) {
+    interceptSearch();
+    cy.visit(standaloneSearchBoxUrl);
+    configure(options);
+  }
+
+  describe('with default option', () => {
+    it('should work as expected', () => {
+      scope('when loading standalone search box', () => {
+        visitStandaloneSearchBox();
+
+        Expect.displayInputSearchBox(true);
+        Expect.displaySearchButton(true);
+        Expect.placeholderContains('Search');
+      });
+
+      scope('when submitting a search', () => {
+        const query = 'test';
+        visitStandaloneSearchBox();
+
+        Actions.typeInSearchBox(query);
+        Expect.displayClearButton(true);
+        Actions.submitSearch();
```

Comment:
for search box, should cover the part when if we press "ENTER" to submit the search